awx/ui/src/contexts/Session.js

```diff
@@ -89,8 +90,15 @@ function SessionProvider({ children }) {
       return () => {};
     }
 
-    const calcRemaining = () =>
-      parseInt(sessionTimeout, 10) - new Date().getTime();
+    const calcRemaining = () => {
```

Comment:
Suggestion: you could also write this as a ternary expression:

```js
const remainingTime = sessionTimeout ? Math.max(...) : 0;
```

Reply:
Thanks @kialam.